### ISSUE TITLE
Check in the notes2md code

### ISSUE THREAD

**Issue by dunhamsteve** [dunhamsteve/notesutils#5]
There is a mostly complete notes2md locally, finish it and check it in.

**Comment by newlegendmedia** [comment]
Yes, please. Also if you could add a quick getting started that explains how to install these tools that would be great. Thanks
